packages/common/src/types/resourceLink.ts

```diff
@@ -8,7 +8,6 @@ export interface ResourceLink {
 
 export interface ResourceLinkSend extends ResourceLink {
```

Comment:
Could we remove this specific ResourceLink as the body is already part of the generic ResourceLink?